crm_event/models/crm_lead.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class CRMLead(models.Model):
+    _inherit = "crm.lead"
+
+    event_type_id = fields.Many2one(
+        comodel_name="event.type",
+        index=True,
+        ondelete="restrict",
+        string="Event type",
+        help=(
+            "If this lead/opportunity is related to a specific event type, "
+            "indicate it here."
+        ),
+    )
```

Comment:
Maybe woulb be good idea add an ondelete??

Reply:
[It's already there](https://github.com/OCA/event/pull/201/files/816ff9125b7c7018c4069ba735912302b97f71a9#diff-6cdd1fe21981658519522db80332ba2dd1899d579ce9b167c5267da1e214057bR13).